pom.xml

```diff
@@ -69,6 +69,7 @@
         <module>test/service-test-utils</module>
         <module>test/test-utils</module>
         <module>test/codegen-generated-classes-test</module>
+        <module>test/sdk-benchmarks</module>
```

Comment:
Hmm, do we want to put it here? This will get released to maven. We can probably just run it locally.

Reply:
Is there a way to build it but not release it?

Reply:
There might be a way but I'm not sure. We can put it here for now and figure out how to exclude the test modules later